app/controllers/bookmarks_controller.rb

```diff
@@ -385,6 +385,7 @@ def bookmark_search_params
       :other_bookmark_tag_names,
       :excluded_bookmark_tag_names,
       rating_ids: [],
+      warning_ids: [],
```

Comment:
Do we need something similar in the works controller, to permit `warning_ids` in `work_search_params`?

Reply:
Yes, good catch, that one escaped.